### DISCUSSION TITLE
Sing-Box/Xray 是共用的同一套规则语法吗？怎么做到的

### DISCUSSION THREAD

**Discussion by chshouyu** [Openwrt-Passwall/openwrt-passwall2#568] [Q&A]
<img width="806" alt="image" src="https://github.com/xiaorouji/openwrt-passwall2/assets/1304342/07f76b47-3dce-478c-92be-484cf7855137">
<img width="797" alt="image" src="https://github.com/xiaorouji/openwrt-passwall2/assets/1304342/86d06bd2-ca9d-47f3-9071-56f69d9d9b02">

sing-box 在新版本中弃用了 `geosite:xxx` 的写法，改为单个的 rule_set 写法，那么在 passwall2 的规则管理界面中的，比如 `geosite:google` 的写法是怎么应用到 sing-box 中的，难道 passwall2 内部做了一次转换？

**Comment by Travel2Here** [comment]
同好奇。反正我用SB不顺畅，总漏。已切换回xray~

**Comment by VergilGao** [comment]
因为是废弃，还没有移除

**Comment by dtknight** [comment]
难怪singbox的时候就有问题，xray就好好的
